diff -urN pavuk-0.9pl27.org/Makefile.am pavuk-0.9pl27/Makefile.am
--- pavuk-0.9pl27.org/Makefile.am	Fri May 11 19:17:31 2001
+++ pavuk-0.9pl27/Makefile.am	Fri May 11 19:17:41 2001
@@ -30,22 +30,12 @@
 	button_icons/old/restart.xpm \
 	button_icons/old/stop.xpm
 
+desktopdir=$(GNOME_PREFIX)/share/gnome/apps/Internet
+desktop_DATA= pavuk.desktop
+
+pixmapsdir=$(GNOME_PREFIX)/share/pixmaps
+pixmaps_DATA=pavuk.png
 dist_head:
 	list="src"; for subdir in $$list; do \
 		(cd $$subdir && $(MAKE) $@); \
 	done
- 
-install-data-local:
-	if test -n "$(DESTDIR)$(GNOME_PREFIX)"; then \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(INSTALL_DATA) $(srcdir)/pavuk.desktop $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-		$(INSTALL_DATA) $(srcdir)/icons/pavuk_64x64.xpm $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-	fi
-
-uninstall-local:
-	if test -n "$(GNOME_PREFIX)"; then \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet/pavuk.desktop; \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps/pavuk_64x64.xpm; \
-	fi
-
diff -urN pavuk-0.9pl27.org/pavuk.desktop pavuk-0.9pl27/pavuk.desktop
--- pavuk-0.9pl27.org/pavuk.desktop	Fri May 11 19:17:31 2001
+++ pavuk-0.9pl27/pavuk.desktop	Fri May 11 19:17:58 2001
@@ -6,6 +6,6 @@
 Comment[sk]=Nástroj na rekurzívne kopírovanie dokumentov z WWW cez protokoly HTTP, HTTPS, FTP, FTPS a Gopher
 Comment[cs]=Nástroj na rekurzivní kopírování dokumentù z WWW pomocí protokolù HTTP, HTTPS, FTP, FTPS a Gopher
 Exec=pavuk -X
-Icon=pavuk_64x64.xpm
+Icon=pavuk.png
 Terminal=0
 Type=Internet
