--- pavuk-0.9pl29d/Makefile.am.orig	Tue Sep 18 23:08:14 2001
+++ pavuk-0.9pl29d/Makefile.am	Tue Feb 12 11:56:19 2002
@@ -48,18 +48,8 @@
 	BZIP2=-9 tar -cjf pavuk-$(VERSION).tar.bz2 pavuk-$(VERSION);\
 	rm -rf pavuk-$(VERSION)
  
+desktopdir=$(GNOME_PREFIX)/share/gnome/apps/Internet
+desktop_DATA= pavuk.desktop
 
-install-data-local:
-	if test -n "$(DESTDIR)$(GNOME_PREFIX)"; then \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(INSTALL_DATA) $(srcdir)/pavuk.desktop $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-		$(INSTALL_DATA) $(srcdir)/icons/pavuk_64x64.xpm $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-	fi
-
-uninstall-local:
-	if test -n "$(GNOME_PREFIX)"; then \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet/pavuk.desktop; \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps/pavuk_64x64.xpm; \
-	fi
-
+pixmapsdir=$(GNOME_PREFIX)/share/pixmaps
+pixmaps_DATA=pavuk.png
diff -urN pavuk-0.9pl27.org/pavuk.desktop pavuk-0.9pl27/pavuk.desktop
--- pavuk-0.9pl27.org/pavuk.desktop	Fri May 11 19:17:31 2001
+++ pavuk-0.9pl27/pavuk.desktop	Fri May 11 19:17:58 2001
@@ -6,6 +6,6 @@
 Comment[sk]=Nástroj na rekurzívne kopírovanie dokumentov z WWW cez protokoly HTTP, HTTPS, FTP, FTPS a Gopher
 Comment[cs]=Nástroj na rekurzivní kopírování dokumentù z WWW pomocí protokolù HTTP, HTTPS, FTP, FTPS a Gopher
 Exec=pavuk -X
-Icon=pavuk_64x64.xpm
+Icon=pavuk.png
 Terminal=0
 Type=Internet
