diff -Nru pavuk-0.9pl26/Makefile.am pavuk-0.9pl26.new/Makefile.am
--- pavuk-0.9pl26/Makefile.am	Thu Jul 13 23:27:26 2000
+++ pavuk-0.9pl26.new/Makefile.am	Tue Oct 17 01:04:11 2000
@@ -30,22 +30,12 @@
 	button_icons/old/restart.xpm \
 	button_icons/old/stop.xpm
 
+desktopdir=$(GNOME_PREFIX)/share/gnome/apps/Internet
+desktop_DATA= pavuk.desktop
+
+pixmapsdir=$(GNOME_PREFIX)/share/pixmaps
+pixmaps_DATA=pavuk.png
 dist_head:
 	list="src"; for subdir in $$list; do \
 		(cd $$subdir && $(MAKE) $@); \
 	done
- 
-install-data-local:
-	if test -n "$(DESTDIR)$(GNOME_PREFIX)"; then \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(INSTALL_DATA) $(srcdir)/pavuk.desktop $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet; \
-		$(mkinstalldirs) $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-		$(INSTALL_DATA) $(srcdir)/icons/pavuk_64x64.xpm $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps; \
-	fi
-
-uninstall-local:
-	if test -n "$(GNOME_PREFIX)"; then \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/gnome/apps/Internet/pavuk.desktop; \
-		rm -f  $(DESTDIR)$(GNOME_PREFIX)/share/pixmaps/pavuk_64x64.xpm; \
-	fi
-
diff -Nru pavuk-0.9pl26/pavuk.desktop pavuk-0.9pl26.new/pavuk.desktop
--- pavuk-0.9pl26/pavuk.desktop	Thu Jul 13 23:31:22 2000
+++ pavuk-0.9pl26.new/pavuk.desktop	Tue Oct 17 01:20:04 2000
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Name=Pavuk
+Name[pl]=Paj±k
 Name[sk]=Pavúk
 Comment=Recursive World Wide Web mirroring tool for HTTP, HTTPS, FTP and Gopher protocols
+Comment[pl]=Narzêdzie do rekursywnego kopiowania zasobów HTTP, HTTPS, FTP i Gopher
 Comment[sk]=Nástroj na rekurzívne kopírovanie dokumentov z WWW cez protokoly HTTP, HTTPS, FTP, FTPS a Gopher
 Exec=pavuk -X
-Icon=pavuk_64x64.xpm
+Icon=pavuk.png
 Terminal=0
 Type=Internet
