--- pavuk-0.9.35/pavuk.desktop~	2010-02-08 00:05:16.000000000 +0200
+++ pavuk-0.9.35/pavuk.desktop	2010-02-08 00:06:25.613115003 +0200
@@ -6,8 +6,9 @@
 Comment[sk]=Nástroj na rekurzívne kopírovanie dokumentov z WWW cez protokoly HTTP, HTTPS, FTP, FTPS a Gopher
 Comment[cs]=Nástroj na rekurzivní kopírování dokumentù z WWW pomocí protokolù HTTP, HTTPS, FTP, FTPS a Gopher
 Comment[de]=Rekursives WWW-Speicherprogramm für die Protokolle HTTP, HTTPS, FTP und Gopher
+Comment[pl]=Program do rekurencyjnego kopiowania stron WWW wspierający protokoły HTTP, HTTPS, FTP, FTPS i Gopher
 Exec=pavuk -X
 Icon=pavuk
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Network;FileTransfer
+Categories=GTK;Network;FileTransfer;
